types/react-redux/index.d.ts

```diff
@@ -164,7 +164,7 @@ interface MergeProps<TStateProps, TDispatchProps, TOwnProps, TMergedProps> {
     (stateProps: TStateProps, dispatchProps: TDispatchProps, ownProps: TOwnProps): TMergedProps;
 }
 
-interface Options extends ConnectOptions {
+interface Options<TStateProps = any, TOwnProps = any, TMergedProps = any> extends ConnectOptions {
```

Comment:
I'm pretty sure that generic arguments are never required in TS - they get inferred by the compiler if they're not provided.

If I'm wrong, I'd prefer the default is to `{}` not to `any` since that's a much safer, more conservative solution.